src/soilbiogeochem/SoilBiogeochemCarbonFluxType.F90

```diff
@@ -708,6 +727,7 @@ subroutine SetValues ( this, num_column, filter_column, value_column)
              i = filter_column(fi)
              this%decomp_cpools_transport_tendency_col(i,j,k) = value_column
              this%decomp_cpools_sourcesink_col(i,j,k)         = value_column  
+             this%decomp_k_col(i,j,k)                         = value_column
```

Comment:
I moved `decomp_k` from the "transitions" loop to the "pools" loop as part of this variable's corrected 3rd dimension.